### PR TITLE
Add support for conv 16x8 int32 bias

### PR DESCRIPTION
**Pull request by paulinesho**
Currently, the 16x8 conv op implicitly assumes int64 biases. This
behavior ignores the converter option
`_experimental_full_integer_quantization_bias_type`. The actual bias
size/type can be retrieved from the model via `GetEvalInput()->type`.
Supporting int32 instead of int64 is expected to lower latency on
average.

Verified that bias->type correctly reflects the converter option set
during conversion. Replicated basic unit tests for int32 biases.

BUG=b/230125424

### CODE REVIEW

File: tensorflow/lite/micro/kernels/conv.cc [hunk 1]
```diff
@@ -74,16 +74,40 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
...
+              tflite::micro::GetTensorShape(filter),
+              tflite::micro::GetTensorData<int8_t>(filter),
+              tflite::micro::GetTensorShape(bias),
+              tflite::micro::GetTensorData<std::int64_t>(bias),
+              tflite::micro::GetTensorShape(output),
+              tflite::micro::GetTensorData<int16_t>(output));
+          break;
+        }
+        default:
+          TF_LITE_KERNEL_LOG(context, "Bias type %s (%d) not supported.",
```
Comment (tingyan19) [inline tensorflow/lite/micro/kernels/conv.cc:107]: MicroPrintf() is preferred than TF_LITE_KERNEL_LOG()

File: tensorflow/lite/micro/kernels/conv_test_common.cc [hunk 2]
```diff
@@ -220,5 +220,23 @@ TfLiteStatus TestConvQuantizedPerChannel(
       conv_params, registration, output_data);
 }
 
+TfLiteStatus TestConvQuantizedPerChannel(
```
Comment (deqiangc) [inline tensorflow/lite/micro/kernels/conv_test_common.cc:223]: Could you please add some comments here? something along the line like "Test XXXX with 16 bit activation, 8 bit weight, 32 bit accumuator".  I would add similiar comments on the other two previous functions. 

This is a style suggestion and I tend to be on verbose side. Feel free to push back.

File: tensorflow/lite/micro/kernels/conv.cc (hunk 1 above)
Comment (paulinesho) [reply tensorflow/lite/micro/kernels/conv.cc]: Fixed TF_LITE_KERNEL_LOG -> MicroPrintf.

File: tensorflow/lite/micro/kernels/conv_test_common.cc (hunk 2 above)
Comment (paulinesho) [reply tensorflow/lite/micro/kernels/conv_test_common.cc:226]: I agree this isn't very readable as is. Added the suggested comment to all 3 functions.

**Review by deqiangc — DISMISSED** [review]

**Review by deqiangc — APPROVED** [review]

**Pull request merged**